subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/LazyDownloadsIntegrationTest.groovy

```diff
@@ -107,10 +107,10 @@ class LazyDownloadsIntegrationTest extends AbstractHttpDependencyResolutionTest
         failure.assertHasCause("Could not resolve test:test:1.0.")
 
         where:
-        expression                                | _
-        "files"                                   | _
-        "fileCollection { true }"                 | _
-        "resolvedConfiguration.resolvedArtifacts" | _
-        "incoming.artifacts"                      | _
+        expression                                                      | _
```

Comment:
It looks like this `where` parameter is ignored by the test?

Reply:
Yeah, this is just a weird way to build a data table in Spock if you only have 1 column you care about.  I'll update it to use a list to get rid of the strangeness.

Reply:
Its not that. The `expression` variable isn't used at all.

[This commit](https://github.com/gradle/gradle/commit/707b2b17cb5c685cc7705d429f4310037ffa8b91) updated the test to not use the variable. We should either update the test again to use the `expression` variable or remove it altogether